Automattic/wp-calypso
# Offer Reset: create Jetpack Cloud section for Pricing page

#### Changes proposed in this Pull Request

* Create a new section for Jetpack Pricing page inside the Jetpack Cloud build process.
* Make this section available for unauthenticated users (behind a feature flag called `jetpack/pricing-page`).
* Include this section in the build process only in dev, stage, and horizon environments.

#### Testing instructions

* Run this PR with Calypso Green build: `yarn start-jetpack-cloud`.
* Open an incognito window.
* Visit `jetpack.cloud.localhost:3000/pricing`.
* Verify that you see a message that says: "Welcome to the Pricing page".

Fixes 1169247016322522-as-1189368424916233
